lib/bitfields.rb

```diff
@@ -90,6 +90,7 @@ def add_bitfield_methods(column, options)
           values = [bitfield_value_was(bit_name), bitfield_value(bit_name)]
           values unless values[0] == values[1]
         end
+        define_method("#{bit_name}_became_true?") { bitfield_value_was(bit_name) != bitfield_value(bit_name) && TRUE_VALUES.include?(bitfield_value(bit_name)) }
```

Comment:
Will fix! @grosser do you prefer I do a follow up commit, or rebase the changes into a single squashed commit?